tests/testthat/custom_tests/regional-dataset-specific.R

```diff
@@ -42,14 +41,14 @@ test_UK_level_1 <- function(region) {
   if (download) {
     test_that(paste(data_name, " downloads sucessfully"), { # nolint
       region$download()
-      expect_s3_class(region$nhs_raw, "data.frame")
-      expect_true(nrow(region$nhs_raw) > 0)
-      expect_true(ncol(region$nhs_raw) >= 2)
+      expect_s3_class(region$data$nhs_raw, "data.frame")
```

Comment:
this should be covered by the new testing if we can get optional features working. Fine for now.